doc/lightning-invoice.7.txt

```diff
@@ -36,6 +36,20 @@ The 'fallbacks' array is one or more fallback addresses to include in
 the invoice (in order from most-preferred to least): note that these
 arrays are not currently tracked to fulfill the invoice.
 
+The 'preimage' is a 64-digit hex string to be used as payment preimage
+for the created invoice.
+By default, if unspecified, lightningd will generate a secure
+pseudorandom preimage seeded from an appropriate entropy source on
+your system.
+*IMPORTANT*: if you specify the 'preimage', you are responsible, to
+ensure appropriate care for generating using a secure pseudorandom
+generator seeded with sufficient entropy, and keeping the preimage
+secret.
+This parameter is intended for use with ZKCP protocols, where the
```

Comment:
This is maybe a bit over specific, mentioning that it is an advanced feature and should not be used unless explicitly needed should suffice here.

Reply:
Okay, will modify.